daml-assistant/integration-tests/src/Main.hs

```diff
@@ -43,10 +44,14 @@ main =
     javaPath <- locateRunfiles "local_jdk/bin"
     mvnPath <- locateRunfiles "mvn_dev_env/bin"
     tarPath <- locateRunfiles "tar_dev_env/bin"
+    -- NOTE: `COMSPEC` env. variable on Windows points to cmd.exe, which is required to be present
+    -- on the PATH as mvn.cmd executes cmd.exe
+    mbComSpec <- getEnv "COMSPEC"
```

Comment:
Oh that looks nice! I wonder if we can use this to avoid the horrible hacks where we call `bash.exe` since `shell` relies on `cmd.exe`.

Reply:
I got rid of it sometime back and I just pushed another commit with a change from `proc` to `shell` for `daml` invocations, which allowed to remove `daml` / `daml.cmd` switch.